src/arrayMethodSort.js

```diff
@@ -4,8 +4,38 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const stringCompare = (a, b) => {
```

Comment:
Choose more appropriate name for function. It should start with a verb.

Reply:
Okay noted. Thank you